content/guides/output-management.md

```diff
@@ -3,85 +3,262 @@ title: Output Management
 
-First let's take a look at where you might stand without these plugins:
+So far we've manually included all our assets in our `index.html` file, but as your application grows and once you start [using hashes in filenames](/guides/caching) and outputting [multiple bundles](/guides/code-splitting-libraries), it will be difficult to keep managing your `index.html` file manually. However, there's no need to fear as a few plugins exist that will make this process much easier to manage.
 
-__index.html__
+## Preparation ##
 
-``` html
-<!doctype html>
+First, let's adjust our project a little bit:
+
+__dist/index.html__
+
+``` diff
+  <html>
+    <head>
+-     <title>Asset Management</title>
++     <title>Output Management</title>
++     <script src="./vendor.bundle.js"></script>
+    </head>
+    <body>
+-     <script src="./bundle.js"></script>
++     <script src="./app.bundle.js"></script>
+    </body>
+  </html>
+```
+
+Now adjust the config. We'll be adding a vendor entry point as an example and we'll change the output as well, so that it will dynamically generate bundle names, based on the entry point names:
+
+__webpack.config.js__
+
+``` diff
+  const path = require('path');
+
+  module.exports = {
+    entry: {
+-     index: './src/index.js',
++     app: './src/index.js',
++     vendor: ['lodash']
+    },
+    output: {
+-     filename: 'bundle.js',
++     filename: '[name].bundle.js',
+      path: path.resolve(__dirname, 'dist')
+    }
+  };
+```
+
+Let's run `npm run build` and see what this generates:
+
+``` bash
+Hash: 5f3b0265b87c603b4a0f
+Version: webpack 2.6.1
+Time: 539ms
+           Asset     Size  Chunks                    Chunk Names
+vendor.bundle.js   544 kB       0  [emitted]  [big]  vendor
+   app.bundle.js  2.81 kB       1  [emitted]         app
+   [0] ./~/lodash/lodash.js 540 kB {0} [built]
+   [1] (webpack)/buildin/global.js 509 bytes {0} [built]
+   [2] (webpack)/buildin/module.js 517 bytes {0} [built]
+   [3] ./src/index.js 172 bytes {1} [built]
+   [4] multi lodash 28 bytes {0} [built]
+```
+
+We can see that webpack generates our `vendor.bundle.js` and `app.bundle.js` files, which we also specified in our `index.html` file. But what would happen if we changed the name of one of our entry points? The generated bundles would be renamed on a build, but our `index.html` file would still reference the old names. Let's fix that with the [`HtmlWebpackPlugin`](/plugins/html-webpack-plugin).
+
+
+## Setting up HtmlWebpackPlugin ##
+
+First install the plugin and adjust the `webpack.config.js` file:
+
+``` bash
+npm install --save-dev html-webpack-plugin
+```
+
+__webpack.config.js__
+
+``` diff
+  const path = require('path');
++ const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+  module.exports = {
+    entry: {
+      app: './src/index.js',
+      vendor: ['lodash']
+    },
++   plugins: [
++     new HtmlWebpackPlugin()
```

Comment:
Two things... I think you need to first show that the old `index.html` file is removed, maybe via a __project__ diff or at least make it clear that it's getting overwritten. The current setup will just overwrite the old one. The second, I think you should pass a title here:

``` js
HtmlWebpackPlugin({
  title: 'Output Management'
})
```